makefile

```diff
@@ -18,7 +18,11 @@ mini-me: mini-me.c
 mini-me.noexec: mini-me
 	cp mini-me $@ && chmod -x $@
 mini-me.setuid: mini-me
-	rm -f $@ && cp mini-me $@&& sudo chown root $@ && sudo chmod u+s $@
+ifneq ($(NO_ROOT),)
```

Comment:
What's `NO_ROOT` ?

Reply:
That's a variable you have to set on the command line. I can drop this commit, but it was useful when testing on a shared server where I don't have root access.

Reply:
I think in that situation it makes sense to just disable mini-me.setuid entirely, not create one that's wrong.

Reply:
I'll drop that change and make a new one that can deal with missing mini-me.setuid.